packages/lab/src/calendar/CalendarNavigation.tsx

```diff
@@ -270,14 +271,16 @@ function OptionWithTooltip<TDate extends DateFrameworkType>({
 
 export const CalendarNavigation = forwardRef<
```

Comment:
Not something can be done now. But if I navigate between month / year dropdown, use arrow to up and down and tab, it selects the month/year for me, which feels a little off. Some ComboBox work was added (#3484)`selectOnTab`, may be needed for Dropdown.

Reply:
one for the community/Salt product radar, if fixed in Dropdown would be picked up by DatePicker.
@ivan-calderon